apps/editor.planx.uk/src/pages/Preview/SingleSession.tsx

```diff
@@ -0,0 +1,80 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Card from "@planx/components/shared/Preview/Card";
+import { CardHeader } from "@planx/components/shared/Preview/CardHeader/CardHeader";
+import { getLocalFlow } from "lib/local";
+import { getLocalFlowIdb } from "lib/local.idb";
+import { useStore } from "pages/FlowEditor/lib/store";
+import React, { PropsWithChildren, useEffect, useState } from "react";
+import { useCurrentRoute } from "react-navi";
+import { Session } from "types";
+import Main from "ui/shared/Main";
+
+export const SingleSession = ({ children }: PropsWithChildren) => {
+  const [
+    flowId,
+    flowName,
+    flowSummary,
+    hasAcknowledgedSingleSessionEntry,
+    setHasAcknowledgedSingleSessionEntry,
+  ] = useStore((state) => [
+    state.id,
+    state.flowName,
+    state.flowSummary,
+    state.hasAcknowledgedSingleSessionEntry,
+    state.setHasAcknowledgedSingleSessionEntry,
+  ]);
+
+  const isContentPage = useCurrentRoute()?.data?.isContentPage;
+
+  const [storedSession, setStoredSession] = useState<Session | undefined>(
+    undefined,
+  );
+  const [loadingSession, setLoadingSession] = useState<boolean>(true);
+
+  useEffect(() => {
+    const loadSession = async () => {
+      try {
+        // First check IndexedDB (preferred storage)
+        //   If flow not in IndexedDB (or db connection fails), try localStorage
+        //   ** Keep in sync with ../src/pages/Preview/Questions `loadSession`
+        const storedSession = await getLocalFlowIdb(flowId);
+        if (!storedSession) getLocalFlow(flowId);
+        setStoredSession(storedSession);
+      } catch (err) {
+        console.log(`Error loading session for flow ${flowId}:`, err);
+      } finally {
+        setLoadingSession(false);
+      }
+    };
+
+    loadSession();
+  }, []);
+
+  return (
+    <>
+      {!loadingSession &&
+      (hasAcknowledgedSingleSessionEntry ||
+        storedSession?.sessionId ||
+        isContentPage) ? (
+        children
+      ) : (
+        <Main>
+          <Card handleSubmit={() => setHasAcknowledgedSingleSessionEntry()}>
+            <CardHeader title={flowName} description={flowSummary}></CardHeader>
+            <Box
+              mb={(theme) => theme.spacing(1)}
+              mt={(theme) => theme.spacing(0.75)}
+            >
+              <Typography variant="subtitle1" component="div">
+                {`You can leave and come back to this service anytime using the same link, device, and browser.`}
```

Comment:
Copy suggestions welcome here! 

Here's an example of before (using Content node) and after:
<img width="1046" height="1085" alt="Screenshot from 2025-12-01 20-23-54" src="https://github.com/user-attachments/assets/2ea48dab-5bd3-475d-80e7-064d40a85af3" />
<img width="1070" height="1087" alt="Screenshot from 2025-12-01 20-23-38" src="https://github.com/user-attachments/assets/225e459d-641f-418e-a25d-bfca2540368e" />

I'm guessing councils won't want to do away with the "this is guidance-only" disclaimer, nor does it make sense to necessarily add to the "summary" - is there a generic way we can include on this page that is less "planning" specific and would work across all non-submission services?